app/controllers/concerns/alchemy/page_redirects.rb

```diff
@@ -0,0 +1,100 @@
+module Alchemy
+  module PageRedirects
+
+    private
+
+    # = Page redirect url
+    #
+    # Lots of reasons exist to redirect to another url, then the requested one.
```

Comment:
...to another URL than the requested one. 

(omit comma, "than" instead of "then"